src/Symfony/Component/HttpFoundation/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 6.4
 ---
 
+ * Add `UriSigner`
```

Comment:
```suggestion
 * Add `UriSigner` from the HttpKernel component
```

Reply:
We also need an entry to HttpKernel's CHANGELOG file (saying it was deprecated, use the one in HttpFoundation instead) and in the `UPGRADE-6.4.md` file in the root of the project (for both components)